### ISSUE TITLE
problem with having the package on the module path

### ISSUE THREAD

**Issue by alexanderkjall** [kohlschutter/junixsocket#49]
I'm trying to use another library (org.testcontainers) that uses this library, and I'm getting this warning:

```
[WARNING] Can't extract module name from junixsocket-native-common-2.0.4.jar: junixsocket.native.common: Invalid module name: 'native' is not a Java identifier
```

**Comment by kohlschuetter** [comment]
This should be fixed in 2.1.0. Please verify. Thanks for reporting!

**Comment by alexanderkjall** [comment]
@kohlschuetter I tried to update the dependency to version 2.1.1 since that was the latest one on maven central and that resolved the problem, thank you.
